model/src/testFixtures/kotlin/io/spine/validation/MessageClass.kt

```diff
@@ -26,28 +26,16 @@
 
 package io.spine.validation
 
-import com.google.protobuf.Descriptors.Descriptor
 import com.google.protobuf.Message
 import kotlin.reflect.KClass
 
-/**
- * Returns a Protobuf descriptor for this message [Class].
- */
-fun Class<out Message>.getDescriptor() =
-    getDeclaredMethod("getDescriptor").invoke(null) as Descriptor
-
-/**
- * Returns a Protobuf descriptor for this message [KClass].
- */
-fun KClass<out Message>.getDescriptor() = java.getDescriptor()
-
 /**
  * Creates a new builder for this message [Class].
  */
-fun Class<out Message>.newBuilder() =
+fun Class<out Message>.newBuilder(): Message.Builder =
```

Comment:
We also have [this thing](https://github.com/SpineEventEngine/base/blob/master/src/main/kotlin/io/spine/protobuf/MessageExts.kt#L77), which caches the results for the repeated calls.

If nothing prevents us from using the utility from `base`, let's do it.

Reply:
I can move this overload to `base`, so it redirects to `builderFor()`.

It just reads more familiar in tests:

<img width="309" alt="image" src="https://github.com/user-attachments/assets/118d5f7f-ced9-4c23-8c8a-2a6b7940a66a" />

Compared to:

<img width="297" alt="image" src="https://github.com/user-attachments/assets/1e7d2413-390d-4078-825e-1d6137a909ec" />